pom.xml

```diff
@@ -104,9 +104,10 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <inceptionYear>2014</inceptionYear>
-        <com.amazonaws.version>1.12.506</com.amazonaws.version>
-        <software.amazon.awssdk.version>2.20.103</software.amazon.awssdk.version>
+        <com.amazonaws.version>1.12.550</com.amazonaws.version>
+        <software.amazon.awssdk.version>2.20.148</software.amazon.awssdk.version>
         <gson.version>2.10.1</gson.version>
+        <io.fabric8.kubernetes.client.version>6.8.1</io.fabric8.kubernetes.client.version>
```

Comment:
This version does not apply to the support branch, but I can remove it in the merge commit.